packages/manager/apps/pci-load-balancer/src/api/hook/useLoadBalancer.tsx

```diff
@@ -178,6 +178,7 @@ export const useAllLoadBalancerListeners = ({
       region,
       'loadbalancer',
       loadBalancerId,
+      'listeners',
```

Comment:
"listeners" should be the first element of the key because it is the largest scope

this way you can invalidate :
- listeners => all listeners
- listeners project projectId => all listeners of a project

if you let "listeners" at the end you cannot invalidate all listeners of a project for example